src/h3lib/lib/baseCells.c

```diff
@@ -863,6 +863,16 @@ void _baseCellToFaceIjk(int baseCell, FaceIJK* h) {
     *h = baseCellData[baseCell].homeFijk;
 }
 
+int _baseCellToCCWrot60(int baseCell, int face) {
```

Comment:
Please add a comment about what this function is doing

Reply:
Will add, thought I had one, but it must have be refactored out.